versions/3.0.3.md

```diff
@@ -3188,7 +3188,7 @@ Field Name | Type | Applies To | Description
 <a name="securitySchemeDescription"></a>description | `string` | Any | A short description for security scheme. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
 <a name="securitySchemeName"></a>name | `string` | `apiKey` | **REQUIRED**. The name of the header, query or cookie parameter to be used.
 <a name="securitySchemeIn"></a>in | `string` | `apiKey` | **REQUIRED**. The location of the API key. Valid values are `"query"`, `"header"` or `"cookie"`.
-<a name="securitySchemeScheme"></a>scheme | `string` | `http` | **REQUIRED**. The name of the HTTP Authorization scheme to be used in the [Authorization header as defined in RFC7235](https://tools.ietf.org/html/rfc7235#section-5.1).  The values used SHOULD be registered in the [IANA Authentication Scheme registry](https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml).
```

Comment:
adding a required statement , might be seen as a breaking change , even if here it is not formulated inside a strong schema validation. so any addition should be set as optionnal 

given the addition should go  in the 3.2 as a target branch 

required statement would mandate a version change , and so going to moonwalk

Reply:
Thank you for the review. Could we possibly change the base branch of the current PR instead of creating a new one? I'll need to check if it has a common history with the current branch to ensure there are no conflicts. Do you think that would be a better approach, or would you recommend creating a new PR?